pages/services/prometheus/0.1.0-2.3.1/troubleshooting/debugging/index.md

```diff
@@ -0,0 +1,198 @@
+---
```

Comment:
the examples used are not from prometheus

Reply:
example referred from nifi as generic steps for dcos debugging